catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -210,14 +211,23 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
         return null;
     }
 
-    protected String buildDerivedImageTitle(String title, String qualifier) {
+    String buildDerivedImageTitle(String title, String qualifier) {
+        String derivedImageTitle;
         String rootFileName = FilenameUtils.getBaseName(title);
 
-        return (StringUtils.isNotBlank(rootFileName) &&
-                StringUtils.isNotBlank(rootFileName.replaceAll("[^A-Za-z0-9]", ""))) // any human readable content
-                ? String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier,
-                        rootFileName.replaceAll(INVALID_FILENAME_CHARACTER_REGEX, ""), JPG)
-                : String.format("%s.%s", qualifier, JPG);
+        // title must contain some alphanumeric, human readable characters, or use default filename
+        if (StringUtils.isNotBlank(rootFileName)
+                && StringUtils.isNotBlank(rootFileName.replaceAll("[^A-Za-z0-9]", ""))) {
```

Comment:
Probably making things more complicated than they need to be. :) The `\W` regex allows underscores in the resulting filename. The check for alphanumeric verifies that after dumping all the alphanumerics, you aren't just left with just an underscore. I assumed that the `\W` could/would be updated in the future to allow more characters, but we would never want to append a bunch of special characters (albeit VALID characters) to the filename; e.g. `original-_.jpg`, or if `+` and `=` were added as valid characters, `original-_=+.jpg`.